curriculum/challenges/english/02-javascript-algorithms-and-data-structures/functional-programming/avoid-mutations-and-side-effects-using-functional-programming.english.md

```diff
@@ -26,10 +26,10 @@ Fill in the code for the function <code>incrementer</code> so it returns the val
 
 ```yml
 tests:
-  - text: Your function <code>incrementer</code> should not change the value of <code>fixedValue</code> (which is <code>4</code>).
-    testString: assert(fixedValue === 4);
   - text: Your <code>incrementer</code> function should return a value that is one larger than the <code>fixedValue</code> value.
     testString: const newValue = incrementer(); assert(newValue === 5);
+  - text: Your function <code>incrementer</code> should not change the value of <code>fixedValue</code> (which is <code>4</code>).
+    testString: assert(fixedValue === 4);
```

Comment:
```suggestion
  - text: Your function <code>incrementer</code> should not change the value of <code>fixedValue</code> (which is <code>4</code>).
    testString: incrementer(); assert(fixedValue === 4);
  - text: Your <code>incrementer</code> function should return a value that is one larger than the <code>fixedValue</code> value.
    testString: const newValue = incrementer(); assert(newValue === 5);
```

This suggestion restores the original order of the tests, per @RandellDawson's comment.